operator/v1/types_csi_cluster_driver.go

```diff
@@ -248,6 +254,17 @@ type GCPCSIDriverConfigSpec struct {
 	KMSKey *GCPKMSKeyReference `json:"kmsKey,omitempty"`
 }
 
+// IBMCloudCSIDriverConfigSpec defines the properties that can be configured for the IBM Cloud CSI driver.
+type IBMCloudCSIDriverConfigSpec struct {
+	// encryptionKeyCRN is the IBM Cloud CRN of the customer-managed root key to use
+	// for disk encryption of volumes for the default storage classes.
+	// +kubebuilder:validation:Required
+	// +kubebuilder:validation:MaxLength:=154
+	// +kubebuilder:validation:MinLength:=144
+	// +kubebuilder:validation:Pattern:=`^crn:v[0-9]+:bluemix:(public|private):(kms|hs-crypto):[a-z-]+:a/[0-9a-f]+:[0-9a-f-]{36}:key:[0-9a-f-]{36}$`
+	EncryptionKeyCRN string `json:"encryptionKeyCRN,omitempty"`
```

Comment:
Should not be omitempty when required
```suggestion
	EncryptionKeyCRN string `json:"encryptionKeyCRN"`
```

Reply:
oops, yep, I'll fix.

Reply:
Updated both items.